crates/web-sys/src/features/gen_AudioContext.rs

```diff
@@ -391,6 +401,10 @@ extern "C" {
     #[doc = "[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/AudioContext/decodeAudioData)"]
     #[doc = ""]
     #[doc = "*This API requires the following crate features to be activated: `AudioContext`*"]
+    #[doc = ""]
+    #[doc = "Argument `success_callback`: See the referenced MDN documentation or the IDL files for the signature of the callback."]
+    #[doc = ""]
+    #[doc = "Return value: While the Promise can produce any JsValue as far as the type system is concerned, practically it is expected to contain a <code>[AudioBuffer]</code>. It can be converted like `<code>let result: [AudioBuffer] = result?.await.into();</code>."]
```

Comment:
I think the `let` here may not be quite right? I don't think Markdown looks inside of backticks to generate links.

Reply:
The backtick is erroneous (now fixed without a regeneration run). The &lt;code&gt; is what allows to have teletypefaced text while still letting markdown look for further syntax to create links.